Tentacle/Issue.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  Issue.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-05-23.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Argo
+import Curry
+
+/// An Issue on Github
+public struct Issue: Hashable, CustomStringConvertible {
+    public enum State: String {
+        case Open = "open"
+        case Closed = "closed"
+	}
+
+    /// The id of the issue
+    public let id: Int
+
+    /// The URL to view this issue in a browser
+    public let url: NSURL?
+
+    /// The number of the issue in the repository it belongs to
+	public let number: Int
+
+    /// The state of the issue, open or closed
+	public let state: State
+
+    /// The title of the issue
+	public let title: String
+
+    /// The body of the issue
+	public let body: String
+
+    /// The author of the issue
+	public let user: User?
+
+    /// The labels associated to this issue, if any
+	public let labels: [Label]
+
+    /// The user assigned to this issue, if any
+	public let assignee: User?
```

Comment:
We should make this `assignees: [User]` since GitHub allows multiple assignees now.